src/sql/src/plan/transform_ast.rs

```diff
@@ -455,9 +455,8 @@ impl<'a> FuncRewriter<'a> {
                 let ident = normalize::ident(ident[0].clone());
                 let fn_ident = match ident.as_str() {
                     "current_role" => Some("current_user"),
```

Comment:
Not related to this PR but I'm curious why we even need this line instead of just adding `current_role` to the union below given we already map `current_role` to `UnmaterializableFunc::CurrentUser => String` in `sql/src/func.rs`